src/main/java/net/sf/jabref/logic/util/Version.java

```diff
@@ -71,17 +101,28 @@ public boolean isNewerThan(Version otherVersion) {
             return false;
         } else if (otherVersion.getFullVersion().equals(BuildInfo.UNKNOWN_VERSION)) {
             return false;
-        } else if (this.getMajor() > otherVersion.getMajor()) {
+        }
+
+        // compare the majors
+        if (this.getMajor() > otherVersion.getMajor()) {
             return true;
         } else if (this.getMajor() == otherVersion.getMajor()) {
+            // if the majors are equal compare the minors
             if (this.getMinor() > otherVersion.getMinor()) {
                 return true;
-            } else {
-                return (this.getMinor() == otherVersion.getMinor()) && (this.getPatch() > otherVersion.getPatch());
+            } else if (this.getMinor() == otherVersion.getMinor()) {
+                // if the minors are equal compare the patch numbers
+                if (this.getPatch() > otherVersion.getPatch()) {
+                    return true;
+                } else if (this.getPatch() == otherVersion.getPatch()) {
+                    // The higher the ordinal the stabler the version (this is the cause the enum has to be sorted)
+                    if (this.developmentStage.ordinal() > otherVersion.developmentStage.ordinal()) {
```

Comment:
I would prefer to introduce an integer which specifies the "degree of stability". People tend to ignore comments even if they stare in their face :-)